sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/src/Customizations/Models/AssetConversionConfiguration.cs

```diff
@@ -19,8 +19,9 @@ public partial class AssetConversionConfiguration
         /// </summary>
         /// <param name="gravity">Gravity vector with respect to object's nominal position.</param>
         /// <param name="scale">Scale of transformation of asset units into meter space.</param>
-        internal AssetConversionConfiguration(System.Numerics.Vector3 gravity, float scale)
-            : this(new Vector3(gravity), scale)
+        /// <param name="disableDetectScaleUnits">Whether or not disable automatic detection of FBX scale units.</param>
+        internal AssetConversionConfiguration(System.Numerics.Vector3 gravity, float scale, bool disableDetectScaleUnits = false)
```

Comment:
I'd still suggest to remove the default value, to make it mandatory. Or does that also cause a failure in generated code?